lib/private/Share20/Manager.php

```diff
@@ -459,7 +459,7 @@ protected function validateExpirationDateInternal(IShare $share) {
 	 * @throws \InvalidArgumentException
 	 * @throws \Exception
 	 */
-	protected function validateExpirationDate(IShare $share) {
+	protected function validateExpirationDateLink(IShare $share) {
```

Comment:
```suggestion
	protected function validateLinkExpirationDate(IShare $share) {
```

if that makes it a tad clearer. just an idea :)

Reply:
the other one is called `validateLinkExpirationDateInternal` so I'd have to rename it also everywhere...
nah... another time, but thanks for the suggestion